### ISSUE TITLE
Rename mboxd dbus object to be in correct xyz.openbmc_project

### ISSUE THREAD

**Issue by geissonator** [openbmc/openbmc#1663]
It's currently in org.openbmc.mboxd which is incorrect.  Will require service rename and minor code updates.

FYI - @anoo1 @dkodihal

**Comment by geissonator** [comment]
Start with https://gerrit.openbmc-project.xyz/#/c/4739/

**Comment by lgon** [comment]
Status Update
End Of Day July 7, 2017
------------------------------
Finished changing dbus name to xyz.openbmc_project.mboxd, and pushed to gerrit for review.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5318> mboxd: Rename mboxd dbus object to xyz.openbmc_project
Resolves  #1663 Rename mboxd dbus object to be in correct xyz.openbmc_project

**Comment by lgon** [comment]
GitHub Comment
++++++++++++++++++++++++++++++++++++++
TASK LIST
- [ ] In phosphor-mboxd repository mplement c++ interfaces defined in .yaml file. https://gerrit.openbmc-project.xyz/#/c/4739/
- [ ] In moboxd repository, implement c interfaces defined in .yaml file.

#STATUS - In development

7-20 - No change in status. Will continue work on this item once #1550 is finished.
++++++++++++++++++++++++++++++++++++++

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/6578> mboxctl: Implemented mboxctl suspend method
Resolves  #1663 Rename mboxd dbus object to be in correct xyz.openbmc_project

**Comment by geissonator** [comment]
As long as we filter org out of REST, this is low priority.

**Comment by anoo1** [comment]
Per Brad, we'll just rename the interface to org.openpower.
